tox.ini

```diff
@@ -33,6 +33,14 @@ setenv =
 commands =
     python -m tests.check_public_interfaces_gpg
 
+# This checks that importing securesystemslib.gpg.constants doesn't shell out on
+# import.
+[testenv:py38-test-gpg-fails]
+setenv =
+    GNUPG = false
+commands =
+    python -c "import securesystemslib.gpg.constants"
```

Comment:
@znewman01, please remove unless necessary.

Reply:
It failed *every time* before, not occasionally.

Note that I set `GNUPG = false` in this environment, which will give an error. That's *different* from `unset GNUPG`, which will find no GPG. 

I don't believe that `py38-no-gpg`  tests the same thing. This test checks "if we shell out when *importing* `gpg.constants` we fail"; `py38-no-gpg` tests "if we have no GPG available, tests still pass."

It might be clearer to replace `false` with a script like `touch /tmp/oops_i_ran` and check that that file wasn't created after import but that felt a little too clever.

As to the question—is this necessary? I don't know; are any tests necessary? Without it, we're not really checking that the issue is fixed—I could revert the rest of the PR and the existing tests would still pass. Up to the maintainers whether you want a test for the fix or not.